### PR TITLE
Font fallbacks

### PR DESCRIPTION
**Pull request by timoxley**
Google blocked in china so the app seems to get stuck waiting for google analytics/fonts.

update: might not be fully *blocked*, but app still blocked for an unreasonable amount of time waiting on fonts.

Took these out of the `head` so they don't block, this might have unwanted "flash of wrong font" effects but I can't really test. Added some fallbacks for both sans and mono fonts. @mondoreale perhaps you can take a look and help me find a good middleground? We could of course consider hosting the font files on our cdn.

If you want to test what this looks like:
* Make sure you disable any locally installed Plex Sans/Mono font
* Disable requests to https://fonts.googleapis.com in devtools
* Reload page without cache.
* Sometimes .css/.pcss files won't rebuild on changing branch, make sure to edit `src/shared/assets/stylesheets/variables.css` and/or `src/shared/assets/stylesheets/index.js`

Might also want to do something similar for https://streamr.network/

### CODE REVIEW

**Comment by timoxley** [comment]
Actually, there might be a better alternative: 
https://chineseseoshifu.com/blog/google-fonts-instable-in-china.html
https://www.hongkiat.com/blog/google-fonts-china/

**Comment by timoxley** [comment]
> host the font there. Then we just use if for both projects.
> Thoughts?

Approved

**Comment by timoxley** [comment]
> host the font there.

One downside is that it does complicate the process of adding new fonts/weights. It'd be good if webpack could take the original `@import`, download the file and rewrite the `@import` as part of the build process, I wonder if something like that exists

**Comment by timoxley** [comment]
> I wonder if something like that exists

✨
 
https://github.com/gabiseabra/google-fonts-webpack-plugin

**Comment by mondoreale** [comment]
Taking over! 💪

**Comment by mondoreale** [comment]
@timoxley I used [`@ibm/plex`](https://github.com/ibm/plex) to get around the google fonts stuff. It goes through webpack, too. There's just less logic around it. What do you think?

**Comment by mondoreale** [comment]
And now the new fonts are also used by storybook. 🔥

**Comment by mondoreale** [comment]
@tumppi can you have a quick look if it works on your end? Please diable IBM fonts in your system and disable cache (in console, maybe) for this and check if there's that flash of fallback font.

**Comment by mondoreale** [comment]
Will resolve conflicts shortly!

File: app/src/editor/shared/components/Sidebar/sidebar.pcss [hunk 1]
```diff
@@ -38,7 +38,7 @@
         margin: 0;
         margin-right: auto;
         color: #323232;
-        font-family: 'IBM Plex Sans', sans-serif;
+        font-family: var(--mono);
```
Comment (mondoreale) [inline app/src/editor/shared/components/Sidebar/sidebar.pcss]: @timoxley I'll use `--sans` here.

File: app/src/marketplace/components/StreamSelector/streamSelector.pcss [hunk 2]
```diff
@@ -102,7 +102,7 @@
   margin: 0;
   background: none;
   cursor: inherit;
-  font-family: 'IBM Plex Sans', sans-serif;
+  font-family: var(--mono);
```
Comment (mondoreale) [inline app/src/marketplace/components/StreamSelector/streamSelector.pcss]: @timoxley, and here, `--sans`.

**Pull request merged**